spearhead/variables.css

```diff
@@ -103,15 +103,15 @@
 		--global--color-primary-hover: #ffffff;
 		--global--color-secondary: #b2b2b2;
 		--global--color-secondary-hover: #cccccc;
-		--global--color-foreground: #b2b2b2;
-		--global--color-foreground-light: #ffffff;
+		--global--color-foreground: #ffffff;
+		--global--color-foreground-low-contrast: #b2b2b2;
 		--global--color-background: #1e1f21;
 		--global--color-text-selection: #000000;
-		--primary-nav--color-background: var(--global--color-foreground-light);
+		--primary-nav--color-background: var(--global--color-foreground-low-contrast);
 		--button--padding-vertical: 24px;
 		--button--padding-horizontal: 32px;
 		--sticky-posts--color-background: var(--global--color-background);
-		--sticky-posts--color-text: var(--global--color-foreground-light);
+		--sticky-posts--color-text: var(--global--color-foreground-low-contrast);
```

Comment:
Should this be `--global--color-foreground`? The dark mode text color for sticky posts used to be `#ffffff`.

Before | This PR
------ | -------
<img width="890" alt="Screen Shot 2020-10-26 at 11 10 14 AM" src="https://user-images.githubusercontent.com/5375500/97190380-0445d100-177c-11eb-9b2f-7bf624552748.png"> | <img width="862" alt="Screen Shot 2020-10-26 at 11 10 31 AM" src="https://user-images.githubusercontent.com/5375500/97190393-07d95800-177c-11eb-97e7-756081cd97f5.png">

Reply:
Yes, but maybe we can remove that line altogether now...

Reply:
I removed these lines as the fallback color is the same.